### PR TITLE
add metrics for replica selection

### PR DESCRIPTION
**Pull request by weiliu1031**
Signed-off-by: Wei Liu <wei.liu@zilliz.com>
issue: #24366
/kind improvement

### CODE REVIEW

**Comment by weiliu1031** [comment]
rerun ut

File: internal/proxy/look_aside_balancer.go [hunk 1]
```diff
@@ -77,10 +80,13 @@ func (b *LookAsideBalancer) Close() {
 }
 
 func (b *LookAsideBalancer) SelectNode(availableNodes []int64, cost int64) (int64, error) {
+	log := log.Ctx(context.TODO()).WithRateGroup("proxy.LookAsideBalancer", 60, 1)
```
Comment (jiaoew1991) [inline internal/proxy/look_aside_balancer.go:83]: need a real context

File: internal/proxy/look_aside_balancer.go [hunk 2]
```diff
@@ -133,6 +144,7 @@ func (b *LookAsideBalancer) calculateScore(cost *internalpb.CostAggregation, exe
 }
 
 func (b *LookAsideBalancer) checkQueryNodeHealthLoop() {
+	log := log.Ctx(context.TODO()).WithRateGroup("proxy.LookAsideBalancer", 60, 1)
```
Comment (jiaoew1991) [inline internal/proxy/look_aside_balancer.go:147]: The same as before

**Pull request merged**